e2e/suites/provider/cases/vault/vault.go

```diff
@@ -19,9 +19,9 @@ import (
 	. "github.com/onsi/ginkgo/v2"
 	v1 "k8s.io/api/core/v1"
 
+	"github.com/external-secrets/external-secrets-e2e/framework"
+	"github.com/external-secrets/external-secrets-e2e/suites/provider/cases/common"
```

Comment:
i replaced a lot of references from `github.com/external-secrets/external-secrets/e2e/***` to `github.com/external-secrets/external-secrets-e2e`.

The issue was that `github.com/external-secrets/external-secrets/e2e` was pinned to a version 0.x.x defined in go.mod; with this we reference the local `e2e` module instead. (thus, changes in `e2e/framework` propagate correctly)